.travis.yml

```diff
@@ -6,6 +6,7 @@ install:
   - pip install -r requirements.txt
   - pip install coveralls
 script:
-  coverage run --source=amy,api,workshops manage.py test
+  - python manage.py makemigrations --dry-run -e; export STATUS_CODE=$?; if [[ "$STATUS_CODE" == "0" ]]; then exit 1; fi
```

Comment:
We can narrow the check to our apps when situation like that happen. If we narrowed it now, we wouldn't know when this situation occurred.